RELEASE.md

```diff
@@ -0,0 +1,359 @@
+Releasing guide
+===============
+
+*(The current process to release a new version still has many manual tasks.
+These should be further automated.)*
+
+This document explains how to release a new version.  As the whole releasing
+process is half automated, this covers the other half humans should manually
```

Comment:
```suggestion
process is half automated, this covers the other half that humans should manually
```
Of course, you can omit this "that", it can be read as `half-humans` (which occurs often in fantasies 😄).